styles.css

```diff
@@ -0,0 +1,55 @@
+body {
+  margin: 20px;
+  font-family: Inter, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 140%;
+  text-align: center;
+  color: #172b4d;
+}
+
+header {
+  margin: 20px;
+}
+
+.flexnav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.landing-page {
+  display: flex;
+  align-items: stretch;
+  flex-direction: column;
+  height: 95vh;
+}
+
+.landing-page > * {
+  flex: 1;
+}
+
+h1 {
+  align-self: center;
+  font-size: 40px;
+  line-height: 52px;
+  margin: 0%;
+}
+
+.enphasis {
+  color: #36b37e;
+}
+
+.network {
+  display: flex;
+}
+
+.social {
+  flex: 1;
+  list-style: none;
+}
```

Comment:
Following the Figma design, Let's try to center the social media icons to improve the UI design. What do you think? 😃 

- This is your design:

<img width="251" alt="image" src="https://user-images.githubusercontent.com/77190203/148441417-f983874f-ad4d-4b57-bfa9-c61bce29bd50.png">